rasa/nlu/featurizers/dense_featurizer/mitie_featurizer.py

```diff
@@ -93,13 +106,30 @@ def _mitie_feature_extractor(self, **kwargs):
             )
         return mitie_feature_extractor
 
-    def features_for_tokens(
+    def _features_as_sequence(
         self,
         tokens: List[Token],
         feature_extractor: "mitie.total_word_feature_extractor",
     ) -> np.ndarray:
-
         vec = []
         for token in tokens:
```

Comment:
what about `cls` token?

Reply:
`cls` token is part of `tokens`, so a corresponding vector is appended automatically.

Reply:
but `feature_extractor.get_feature_vector(token.text)` would return some gibberish in this case for `__CLS__`